cloud_experts_tutorials/cloud-experts-rosa-with-hcp-private-offer-acceptance-and-sharing.adoc

```diff
@@ -0,0 +1,187 @@
++
+.Private offer acceptance confirmation window
++
+image::rosa-private-offer-acceptance-confirmation-window.png[]
+
+. If the accepted private offer service start date is set to be immediately following the offer acceptance, you can click the *Set up your account* button in the confirmation modal window shown in Figure 8.
++
+.Subscription confirmation
++
+image::rosa-subscription-contfirmation.png[]
+
+. If the accepted private offer has a future start date specified, return to the private offer page after the service start date, these are always in UTC time zone, and click the *Setup your account* button to proceed with the Red{nbsp}Hat and AWS account linking. With no agreement active, the account linking described below is not triggered, the _Account setup_ process can be done only after the _Service start date_.
+
+== Sharing a private offer
+
+. Clicking the *Set up your account* button in the previous step takes you to the AWS and Red{nbsp}Hat account linking step. At this time, you are already logged in with the AWS account that accepted the offer. If you are not logged in with a Red{nbsp}Hat account, you will be prompted to do so.
++
+{hcp-title} entitlement is shared with other team members through your Red{nbsp}Hat organization account. All existing users in the same Red{nbsp}Hat organization are able to select the billing AWS account that accepted the private offer by following the above described steps. You can link:https://www.redhat.com/wapps/ugc/protected/usermgt/userList.html[manage users in your Red{nbsp}Hat organization], when logged in as the Red{nbsp}Hat organization administrator, and invite or create new users.
++
+[NOTE]
+====
+{hcp-title} private offer cannot be shared with AWS linked accounts through the AWS License Manager.
+====
+
+. Check link:https://access.redhat.com/customer-service-users[this user management FAQ] for more details about Red{nbsp}Hat account user management tasks. Make sure that the already logged in Red{nbsp}Hat account includes all users that are meant to be ROSA cluster deployers benefiting from the accepted private offer.
+
+. Make sure that the Red{nbsp}Hat account number and the AWS account ID are the desired accounts that are to be linked. This linking is unique and a Red{nbsp}Hat account can be connected only with a single AWS (billing) account.
++
+.AWS and Red{nbsp}Hat accounts connection
++
+image::rosa-aws-and-red-hat-accounts-connection.png[]
+
+. Click *Connect accounts* after you are confident that the correct Red{nbsp}Hat account and AWS billing account are being connected . The *Connect accounts* button is available only after agreeing to the terms and conditions.
+
+. If you would like to link the AWS account with another Red{nbsp}Hat account than is shown on this page in Figure 9, log out from the Red{nbsp}Hat Hybrid Cloud Console before connecting the accounts and repeat the step of setting the account by returning to the private offer URL that is already accepted.
```

Comment:
Should this go above step 4? Because at this point they've already connected the accounts